apps/client/src/app/shared/code-system.service.ts

```diff
@@ -11,8 +11,8 @@ export class CodeSystemService {
 
   }
 
-    public save(codeSystem: CodeSystem): Observable<CodeSystem> {
-        if (codeSystem.id) {
+    public save(codeSystem: CodeSystem, id?: String): Observable<CodeSystem> {
+        if (id) {
```

Comment:
What is the value of the `id` argument here? We don't do anything with it other than check its existance in `if (id) {`...

Reply:
Same as above.

Reply:
Removed